src/media/mod.rs

```diff
@@ -208,25 +217,24 @@ impl ::std::fmt::Display for MediaCategory {
     }
 }
 
-/// A builder struct that allows you to build up parameters to a media upload before initiating it.
+/// A builder that provides mechanism to create upload and customize it.
 ///
-/// `UploadBuilder` is your entry point to uploading media to Twitter. This allows you to configure
-/// an upload and set the proper metadata, so that when you `call` it to begin the upload proper,
-/// the resulting [`UploadFuture`] can take care of all the underlying details of the upload itself.
+/// `UploadBuilder` is entry point to uploading media to Twitter.
+///  It allows you to configure an upload and set the proper metadata.
 ///
 /// [`UploadFuture`]: struct.UploadFuture.html
 ///
 /// To begin setting up an upload, call `new` with your data and its media type. (Convenience
 /// functions to create `Mime` instances for types Twitter is known to accept are available in the
-/// [`media_types`] module.) After that, you can configure the upload, and finally start the
-/// process with `call`. A basic example of using `UploadBuilder` to upload an image and attach it
+/// [`media_types`] module.) With that, you can configure the upload, and finally start the
+/// process using `call` method. See a basic example of using `UploadBuilder` to upload an image and attach it
 /// to a Tweet is in [the module documentation].
 ///
 /// [`media_types`]: media_types/index.html
 /// [the module documentation]: index.html
 ///
 /// To see more precise specifications for what media formats Twitter supports (resolution, file
-/// size, etc), see [their API documentation][media-best-practices]. Note that `UploadBuilder`
+/// size, etc), see [the API documentation][media-best-practices]. Note that `UploadBuilder`
```

Comment:
The reason i use "their" in this sentence is to differentiate between API documentation for *egg-mode* (i.e. that i wrote) and API documentation for *Twitter itself* (i.e. that i did not write). I would prefer to keep that distinction here.

Reply:
ok